### DISCUSSION TITLE
Error during importing a script

### DISCUSSION THREAD

**Discussion by platinumindustries** [denoland/deno#10949] [Q&A]
I Have posted this question on [SO](https://stackoverflow.com/questions/67939075/importing-couchnode-as-esm-gives-error-error-jsdoc-augments-is-not-attach). I'm trying to import the **_couchbase nodejs client_** using **_jspm_** package manager/CDN but typescript keeps on complaining about **_TypeDoc_** params. Any help is appreciated.

I also cant use the nodejs compatibility layer as some libraries arent finished and it brings up more errors than just converting the package to ESM via jspm.

**Comment by wperron** [comment]
importing npm packages through a CDN is a bit of an uneven experience -- each CDN does bits of transformations differently than the others. Some packages might work one CDN but not on another, while some packages simply might not work at all. My recommendation would be to try https://skypack.dev and https://esm.sh as well and see if that helps.

Alternatively, since this seems to be an error during the type checking, you could also try running with `--no-check` and see if that helps -- of course then you won't have any type checking in effect.

**Reply by platinumindustries** [reply]
can i select the --no-check option only on that one script that i'm importing?
